tbans/controllers/notification_controller.py

```diff
@@ -0,0 +1,91 @@
+import logging
+import tba_config
+import webapp2
+
+from tba.consts.notification_type import NotificationType
+
+from tbans.consts.vertical_type import VerticalType
+
+
+class NotificationHandler(webapp2.RequestHandler):
+
+    def post(self):
+        notification_type = int(self.request.get('notification_type'))
+
+        user_id = self.request.get('user_id', None)
+        if user_id:
+            sending_device_key = self.request.get('sending_device_key', None)
+            notification = NotificationHandler._user_notification(user_id, notification_type)
+            # We know if we're sending to users, we'll only be sending to mobile clients
+            from tba.models.mobile_client import MobileClient
+            tokens = MobileClient.device_push_tokens(user_id)
+            # If we have a `sending_device_key` - remove it from our list of tokens
+            # do we don't notify the client that dispatched an update to other clients
+            if sending_device_key in tokens:
+                tokens.remove(sending_device_key)
+            # If we don't have any devices to send to - gracefully return
+            if len(tokens) == 0:
+                return
+        else:
+            # TODO: Handle notification types
+            pass
+
+        vertical_type = int(self.request.get('vertical_type'))
+        if vertical_type == VerticalType.FCM:
+            success = NotificationHandler._send_fcm(notification, tokens)
+            # If our FCM send failed, requeue to try again
+            if not success:
+                self.response.set_status(500)
+        elif vertical_type == VerticalType.WEBHOOK:
+            # TODO: Dispatch to webhooks
```

Comment:
- [ ] Dispatch notifications to webhooks (this code shouldn't execute for myTBA notifications)

Reply:
Going to table this for now, since we don't need this for sending myTBA notifications